10up/classifai
# Add support controlling feature availability

### Description of the Change
PR adds an enable/disable feature setting option for the below features. PR also improves the codebase for better maintainability.

- Language Processing / IBM Watson / Natural Language Understanding
- Language Processing / Microsoft Azure / Text to Speech
- Recommended Content / Microsoft Azure / AI Personalizer

Closes #589 

**Note for the reviewer:**
To avoid merge conflict I have started work from [enhancement/588](https://github.com/10up/classifai/tree/enhancement/588) branch and kept it as base branch for this PR.

### How to test the Change
1. Go to Classifai settings for the specific feature.
1. Toggle feature availability for the below features and verify that feature availability reflects accordingly on site.
- Language Processing / IBM Watson / Natural Language Understanding
- Language Processing / Microsoft Azure / Text to Speech
- Recommended Content / Microsoft Azure / AI Personalizer

### Changelog Entry
> Added - Enable/disable toggle option for the "Classify content" feature.
> Added - Enable/disable toggle option for the "Text to Speech" feature.
> Added - Enable/disable toggle option for the "Recommended content" feature.

### Credits
Props @iamdharmesh @jeffpaul 


### Checklist:
